client/src/pages/Login.jsx

```diff
@@ -20,14 +22,14 @@ function useQuery() {
 /**
  * Displays the login page over everything else
  */
-const Login = () => {
+const Login = (): Node => {
   const didLoginFail = useQuery().get(LOGIN_FAILURE_QUERY_PARAM);
 
   return (
     <div className="login-wrapper">
       <div className="login-card">
         <h2>Member Database Login</h2>
-        {didLoginFail && (
+        {didLoginFail !== null && didLoginFail !== undefined && (
```

Comment:
Curious about this change, I thought what was previously there was accomplishing the same thing more succinctly?

Reply:
This avoids [sketchy-null](https://flow.org/en/docs/linting/rule-reference/#toc-sketchy-null) check. We want to avoid this because we want to specifically avoid [complicated matching behavior](https://262.ecma-international.org/5.1/#sec-11.9.3), especially when enforcing type.